src/firefly/java/edu/caltech/ipac/table/io/VoTableReader.java

```diff
@@ -368,24 +368,39 @@ private static void getTableElements(VOElement root, List<TableElement>tblAry) {
             getTableElements(top, tableAry);
 
             if (tableAry.size() == 0) {
-                // check for errors: section 4.4 of http://www.ivoa.net/documents/DALI/20170517/REC-DALI-1.1.html
-                VOElement[] resources = getResourceChildren(top);
-                for (VOElement r : resources) {
-                    if ("results".equals(r.getAttribute("type"))) {
-                        VOElement[] infos = r.getChildrenByName("INFO");
-                        for (VOElement info : infos) {
-                            if ("QUERY_STATUS".equals(info.getName()) && "ERROR".equals(info.getAttribute("value"))) {
-                                throw new DataAccessException(info.getTextContent());
-                            }
-                        }
-                    }
-                }
+                checkForErrors(top);
             }
         }
 
         return tableAry;
     }
 
+    private static void checkForErrors(VOElement top) throws DataAccessException {
```

Comment:
This looks like a TAP's specific error condition.  I am not sure it should be here.  Your thoughts?

Reply:
> There is lots of space. Why not put the label outside the fields:

I encouraged @tgoldina to keep it inside for now because I would like to look at ways to make the whole upper area smaller to leave more rooms for the table on bottom.  Of course we are going to have to figure out how to make the spacial input smaller as well.

To keep inside it will have to stand out more and It cannot scroll, it needs to stay like a title area. Not scrolling is important. If the widget can't do that then we should just go ahead and move it out.